store/populations.go

```diff
@@ -28,11 +28,26 @@ import (
 	pb "github.com/datacommonsorg/mixer/proto"
 	"github.com/datacommonsorg/mixer/util"
 	"github.com/golang/protobuf/jsonpb"
-	"github.com/golang/protobuf/proto"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/api/iterator"
 )
 
+type rankKey struct {
+	prov    string
+	mmethod string
+}
+
+var statsRanking = map[rankKey]int{
+	rankKey{"CensusPEP", "CensusPEPSurvey"}:                   0,
+	rankKey{"CensusACS5YearSurvey", "CensusACS5yrSurvey"}:     1,
+	rankKey{"EurostatData", "EurostatRegionalPopulationData"}: 2,
+	rankKey{"WorldDevelopmentIndicators", ""}:                 3,
+	rankKey{"BLS_LAUS", "BLSSeasonallyUnadjusted"}:            0,
+	rankKey{"EurostatData", ""}:                               1,
```

Comment:
Add some comments on what the ranking means?

Also, add inline comments to demarcate
  // Population
  // Unemployment Rate
?

Reply:
Done